registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/status/util/StatusUtil.java

```diff
@@ -152,7 +152,7 @@ public enum StatusUtil {
 	LINK_RID_FOR_LOST_PACKET_FAILED(StatusConstants.UIN_GENERATOR_MODULE_FAILED + "008","UIn not found the the matched RID"),
 	UIN_ALREADY_EXIST_IN_IDREPO(StatusConstants.UIN_GENERATOR_MODULE_FAILED + "009", "Received ID record already exists error from idrepo even after trying with force merge"),
 	OLD_APPLICATION_ID(StatusConstants.UIN_GENERATOR_MODULE_FAILED + "010","Received error as old application ID reprocessing not allowed from idrepo"),
-	
+	INVALID_INPUT_PARAMETER(StatusConstants.UIN_GENERATOR_MODULE_FAILED+"011","Recived error as Invalid_Input_Parameter"),

```

Comment:
Please update Received error as Invalid Input Parameter and do the spell check

Reply:
Done.